Source/Blazorise/Components/Droppable/DropZone.razor

```diff
@@ -5,18 +5,51 @@
     <div @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" @ondragenter="@OnDragEnterHandler" @ondragleave="@OnDragLeaveHandler" @ondrop="@OnDropHandler" @attributes="@Attributes">
         @ChildContent
 
-        @foreach ( var item in GetItems() )
+        @if ( OnlyZone != true )
```

Comment:
!OnlyZone 
Matter of personal preference and Blazorise prefered code convention I guess.